lib/dpul_collections/indexing_pipeline/figgy/resource.ex

```diff
@@ -140,6 +153,28 @@ defmodule DpulCollections.IndexingPipeline.Figgy.Resource do
     |> Map.new()
   end
 
+  defp remove_non_displayable_filesets(resources) do
+    resources
+    |> Enum.reject(fn r -> removable_resource?(r) end)
+  end
+
+  defp removable_resource?(%__MODULE__{metadata: %{"file_metadata" => file_metadata}}) do
```

Comment:
I think technically this is going to fail for resources like https://figgy.princeton.edu/catalog/eb7ae5f9-cf1f-4594-93b3-25e8aa34055a (recordings with an image of the physical tape), but I don't think we have any Ephemera items like that, we can cross that bridge when we get there.

Reply:
My hunch is that we'll be indexing and displaying audio files by the the time we get to these.